JobParseAndNormalize.md

```diff
@@ -0,0 +1,353 @@
+Job Parse and Normalize API
+====================
+
+####Table of Contents
+_______
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+- [Response structure](#response-structure)
+- [Versioning](#versioning)
+
+##Summary
+
+The Job Parse and Normalize (JPAN) service parses a Base64-encoded job posting and further enriches the parsed data with the following normalizations and classifications.  
+
+ #####Enrichments:
+ - Job Title Classifications
+    - [Carotene](https://github.com/cbdr/DataScienceAPIDocumentation/blob/master/JobTitle.md) <sup>(optional)</sup>
+    - [ONET](https://github.com/cbdr/DataScienceAPIDocumentation/blob/master/JobTitle.md) <sup>(optional)</sup>
+    - Textkernel
+ - [Normalized Company Name](https://github.com/cbdr/DataScienceAPIDocumentation/blob/master/CompanyNormalization.md)  <sup>(optional)</sup>
+ - Skills Extraction
+    - [Skills 4.0](https://github.com/cbdr/DataScienceAPIDocumentation/blob/master/Skills.md) <sup>(optional)</sup>
```

Comment:
not sure we should bother with naming versions, as this will change over time (and probably fall out of date when we forget to update this). the important part IMO is "the job parsing API extracts skills."